package.json

```diff
@@ -22,8 +22,8 @@
 		"watch": "NODE_ENV=development webpack --progress --watch --config webpack.js",
 		"lint": "eslint --ext .js,.vue src",
 		"lint:fix": "eslint --ext .js,.vue src --fix",
-		"stylelint": "stylelint src",
-		"stylelint:fix": "stylelint src --fix",
+		"stylelint": "stylelint {src,css}/**/*.{html,vue,css,sass,scss,less}",
+		"stylelint:fix": "stylelint {src,css}/**/*.{html,vue,css,sass,scss,less} --fix",
```

Comment:
Could be reverted again after https://github.com/nextcloud/stylelint-config/pull/22 is released